spark/src/main/scala/org/apache/comet/CometSparkSessionExtensions.scala

```diff
@@ -1399,6 +1404,16 @@ object CometSparkSessionExtensions extends Logging {
     }
   }
 
+  def cometUnifiedMemoryManagerEnabled(sparkConf: SparkConf): Boolean = {
+    sparkConf.getBoolean("spark.memory.offHeap.enabled", false)
+  }
+
+  def cometShuffleUnifiedMemoryManagerEnabled(sparkConf: SparkConf): Boolean = {
```

Comment:
Perhaps it is best to add `Test` into the name of this function to indicate this is only for test purpose

Reply:
thank you, added